go.mod

```diff
@@ -3,9 +3,11 @@ module fyne.io/fyne/v2
 go 1.12
 
 require (
+	fyne.io/fyne v1.4.3
```

Comment:
This seems wrong, I think. We shouldn’t need to require an old version of fyne in the go.mod. There is probably a stray import somewhere that is using the old import path. Might be worth running `go mod tidy` a second time.